scala/spark-wordcount/pom.xml

```diff
@@ -9,11 +11,12 @@
     <version>0.1</version>
 
     <properties>
-        <bigtable.version>1.7.0</bigtable.version>
-        <hbase.version>1.3.1</hbase.version>
+        <bigtable.version>1.15.0</bigtable.version>
+        <hbase.version>1.5.0</hbase.version>
+        <hadoop.version>2.10.0</hadoop.version>
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.source>1.8</maven.compiler.source>
-        <spark.version>2.4.0</spark.version>
+        <spark.version>2.4.5</spark.version>
         <scala.version>2.11.12</scala.version>
```

Comment:
can we use scala 2.12.10 here?

Reply:
(To track dataproc 1.5.x)

Reply:
Checking it out...

Reply:
Caution: "Spark 2.x is pre-built with Scala 2.11" (http://spark.apache.org/downloads.html) so we'd have to change the steps to install Spark 2.x with Scala 2.12 locally that is not trivial (you have to build it yourself). I'd skip it since Scala 2.12 supports 2.11. OK?

Reply:
Please update your comment above to note this

Reply:
done